target/linux/mpc85xx/image/p1020.mk

```diff
@@ -1,20 +1,37 @@
-define Device/aerohive_hiveap-330
+define Device/aerohive_hiveap
   DEVICE_VENDOR := Aerohive
-  DEVICE_MODEL := HiveAP-330
   DEVICE_PACKAGES := kmod-tpm-i2c-atmel
   BLOCKSIZE := 128k
   KERNEL := kernel-bin | gzip | uImage gzip
-  KERNEL_SIZE := 8m
   KERNEL_INITRAMFS := copy-file $(KDIR)/vmlinux-initramfs | uImage none
-  IMAGES := fdt.bin sysupgrade.bin
   IMAGE/fdt.bin := append-dtb
+  IMAGES := fdt.bin sysupgrade.bin kernel.bin rootfs.bin
+  IMAGE/kernel.bin := append-kernel
```

Comment:
I added these two additional image targets (`rootfs.bin` and `kernel.bin`) because I thought they might have some magic effect to building the `sysupgrade-bin`.

At this point I don't believe this is true. I will remove and re-test.

Reply:
Cleaned up in commit https://github.com/Hurricos/openwrt/commit/75ce435307b25a19fb50efcf1534eaebed6843eb.